SECURITY.md

```diff
@@ -16,29 +16,93 @@ You may need to [update the security provider](https://developer.android.com/tra
 
 ## TLS with OpenSSL
 
-This is currently the recommended approach for using gRPC over TLS (on non-Android systems). 
+This is currently the recommended approach for using gRPC over TLS (on non-Android systems).
 
-### Benefits of using OpenSSL
+The main benefits of using OpenSSL are:
 
 1. **Speed**: In local testing, we've seen performance improvements of 3x over the JDK. GCM, which is used by the only cipher suite required by the HTTP/2 spec, is 10-500x faster.
 2. **Ciphers**: OpenSSL has its own ciphers and is not dependent on the limitations of the JDK. This allows supporting GCM on Java 7.
 3. **ALPN to NPN Fallback**: if the remote endpoint doesn't support ALPN.
 4. **Version Independence**: does not require using a different library version depending on the JDK update.
 
-### Requirements for using OpenSSL
+Support for OpenSSL is only provided for the Netty transport via [netty-tcnative](https://github.com/netty/netty-tcnative), which is a fork of
+[Apache Tomcat's tcnative](http://tomcat.apache.org/native-doc/), a JNI wrapper around OpenSSL. As of version `1.1.33.Fork13`, netty-tcnative provides two options for usage: statically or dynamically linked. For simplification of setup,
+we recommend using `netty-tcnative-boringssl-static`, which is statically linked against BoringSSL and Apache APR. Using this artifact requires no extra installation and guarantees that ALPN and the ciphers required for
```

Comment:
Netty-tcnative-boringssl-static is certainly more convenient for initial setup, but using it has security implications that GRPC users might not grasp. Users who follow the recommendation are more likely to be vulnerable. Perhaps the recommendation should be more nuanced? It's your call, of course...